src/transactions/feed/SwapFeedItem.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import SwapIcon from 'src/icons/SwapIcon'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { useTokenInfo } from 'src/tokens/hooks'
+import { TokenExchange } from 'src/transactions/types'
+
+interface Props {
+  exchange: TokenExchange
+}
+
+function SwapFeedItem({ exchange }: Props) {
+  const { t } = useTranslation()
+  const incomingTokenInfo = useTokenInfo(exchange.inAmount.tokenAddress)
+  const outgoingTokenInfo = useTokenInfo(exchange.outAmount.tokenAddress)
+
+  const openTransferDetails = () => {
+    navigate(Screens.TransactionDetailsScreen, { transaction: exchange })
+    ValoraAnalytics.track(HomeEvents.transaction_feed_item_select)
+  }
+
+  return (
+    <Touchable disabled={false} onPress={openTransferDetails}>
+      <View style={styles.container}>
+        <View style={styles.iconContainer}>
+          <SwapIcon />
+        </View>
+        <View style={styles.contentContainer}>
+          <Text style={styles.title} testID={'SwapFeedItem/title'}>
+            {t('swapScreen.title')}
+          </Text>
+          <Text style={styles.subtitle} testID={'SwapFeedItem/subtitle'}>
+            {outgoingTokenInfo?.symbol} {'to'} {incomingTokenInfo?.symbol}
```

Comment:
we should localise this, and not hardcode "to". perhaps better to localise the whole thing, so the translation value can be "{{token1}} to {{token2}}"

Reply:
I added localization!